libqtile/widget/window_count.py

```diff
@@ -37,10 +37,11 @@ class WindowCount(base._TextBox):
         ("show_zero", False, "Show window count when no windows")
     ]  # type: List[Tuple[str, Any, str]]
 
-    def __init__(self, text=" ", width=bar.CALCULATED, **config):
+    def __init__(self, text=" ", width=bar.CALCULATED, this_screen=False, **config):
```

Comment:
This shouldn't be an option in `__init__`. Any additional parameters should be added to the `defaults` list.

Reply:
That said, I don't think we need an option here as we should just fix the behaviour.

Reply:
Okay, thanks for the feedback, then I'll remove the option and the if statements.

Reply:
Can you also expand the docstring for the widget to explain it shows the window count for the relevant screen too,

Also, ideally, we'd add a test for this on multiple screens. Let me know if you need help with that.

Reply:
I improved the docstring.
I tried to implement the tests, but somehow I always get the error `test_window_count[1-x11] - AssertionError: Window never appeared...`
I don't know if I made some mistake in setting up my test environment.
But the other test with the Attribute Errors works and I already fixed an uncaught exception bug there.

Reply:
Thanks. I'm out for a bit but will take a look later.

Reply:
I gave you permission to push directly on the branch of my fork. I think this will make it much easier, if you want to contribute to it.